tests/post/steps/test_service_configuration.py

```diff
@@ -238,15 +261,35 @@ def load(self, full_csc, name, namespace):
             csc_obj = yaml.safe_load(full_csc.data['config.yaml'])
         except yaml.YAMLError as exc:
             raise Exception(
-                'Invalid YAML format in ConfigMap {} found in namespace {}: {!s}'
-                .format(name, namespace, exc)
+                'Invalid YAML format in ConfigMap {} found in namespace {}: '
+                '{!s}'.format(name, namespace, exc)
             )
         except Exception as exc:
             raise Exception(
-                "Failed loading `config.yaml` from ConfigMap {} in namespace {}: "
-                "{!s}".format(name, namespace, exc)
+                "Failed loading `config.yaml` from ConfigMap {} in namespace "
+                "{}: {!s}".format(name, namespace, exc)
             )
         return csc_obj
 
 
+def _query_prometheus_api(host, route):
+    cp_ip = _get_local_grain(host, 'metalk8s:control_plane_ip')
+
+    return requests.get(
+        'https://{ip}:8443/api/prometheus/api/v1/{route}'
+        .format(ip=cp_ip, route=route),
+        verify=False,
+    )
+
```

Comment:
We should refactor this particular method and the following to reduce duplication.
We have the same method in `test_monitoring.py`

We could keep them in  `utils` then call them from there just like `utils.get_dict_element`